hawkular-bus/hawkular-bus-common/pom.xml

```diff
@@ -37,6 +37,18 @@
   </properties>
 
   <dependencies>
+    <!-- To avoid javadoc errors (and thus causing mvn release plugin to fail) we need this provided dependency.
```

Comment:
Is this really needed? I've seen similar comments on the `hawkular/modules`, and the javadoc passed/failed regardless of the dependency.